server/src/routers/FavoritesRouter/addFavorite.js

```diff
@@ -15,6 +15,9 @@ export default async function addFavorite(token, menuItemID) {
         await firestore.doc("favorites/users").update({
             [token]: firebase.firestore.FieldValue.arrayUnion(menuItemID)
         });
+        await firestore.doc("menus/menuItems").update({
```

Comment:
Nice - I like the idea of making a new collection for this!

Reply:
Yeah, realized that was the best option about 30s after our conversation :)